snippets/test-intro-zoe.js

```diff
@@ -0,0 +1,140 @@
+// WARNING: THE CODE IN THIS FILE WILL BE SHOWN ON THE AGORIC
+// DOCUMENTATION SITE.
+
+import '@agoric/install-ses';
+import test from 'ava';
+import { E } from '@agoric/eventual-send';
+import { makeZoe } from '@agoric/zoe';
+import { makeIssuerKit } from '@agoric/ertp';
+import bundleSource from '@agoric/bundle-source';
+import { makeFakeVatAdmin } from './tools/fakeVatAdmin';
+
+// #region importBundleSource
+// #endregion importBundleSource
+
+test('intro to zoe', async t => {
+  const fakeVatAdmin = makeFakeVatAdmin();
+
+  const zoe = makeZoe(fakeVatAdmin);
+
+  const moolaKit = makeIssuerKit('moola');
+  const simoleanKit = makeIssuerKit('simoleans');
+
+  const purse = simoleanKit.issuer.makeEmptyPurse();
+  purse.deposit(
+    simoleanKit.mint.mintPayment(simoleanKit.amountMath.make(1000)),
+  );
+  const aliceProposal = harden({
+    give: { Asset: moolaKit.amountMath.make(3) }, // asset: 3 moola
+    want: { Price: simoleanKit.amountMath.make(7) }, // price: 7 simoleans
+  });
+  const alicePayments = harden({
+    Asset: moolaKit.mint.mintPayment(moolaKit.amountMath.make(3)),
+  });
+
+  const atomicSwapFile = `${__dirname}/../node_modules/@agoric/zoe/src/contracts/atomicSwap`;
+
+  // #region bundle
```

Comment:
Rather than having a really flat namespace, would it make sense to prefix the label with the documentation section or page that the code is for?

```suggestion
  // #region intro-zoe-bundle
```

Reply:
I think we shouldn't put that information in the region, since the snippet import itself includes the filename. I think each doc should have a different coe file.

Reply:
> each doc should have a different code file.

In that case, my suggestion is unnecessary.